www/main/banidb/realm-search.js

```diff
@@ -62,7 +62,7 @@ const query = (searchQuery, searchType, searchSource) =>
     // default source for ang search to GURU_GRANTH_SAHIB
     let angSearchSourceId = CONSTS.SOURCE_TYPES.GURU_GRANTH_SAHIB;
     const order = [];
-    let howManyRows = 20;
+    let howManyRows = howManyRowsToShow;
```

Comment:
@theamanjs veerji 
where are we using this variable, can't we use the state directly?

Reply:
Veerji, I'm using this because we cannot mutate the argument passed to a function. Eslint would throw error on param reassign.

The variable is being reassigned a value here https://github.com/KhalisFoundation/sttm-desktop/blob/2f91ec203077cfe7c5d010d3fb4ef75741607249/www/main/banidb/realm-search.js#L131